projects/sample/initialize.py

```diff
@@ -1,57 +1,119 @@
+    to the cloud that you will need for your entire scanning session.
+
+    In this demo, things that are on the console computer will be in the folder
+    'on_console'. We will transfer files to the cloud (e.g., 'on_cloud'). Here,
+    everything is on the same computer but this doesn't have to be the case
+    when you run your own experiment!
+
+    INPUT:
+        [1] cfg (configuration file with important variables)
+        [2] fileInterface (this will allow you to call useful functions)
+        [3] projectComm (communication pipe to talk with projectInterface)
+    OUTPUT:
+        None.
+    """
+
+    # define directories where files are on the console ('tmp/console_directory/')
+    #   and where files are on the cloud ('tmp/cloud_directory')
+    consoleDir = os.path.join(currPath,'tmp/console_directory/')
+    cloudDir = os.path.join(currPath,'tmp/cloud_directory/tmp/')
+    # here, we will actually make the directories!
+    try:
+        os.makedirs(consoleDir)
+        os.makedirs(cloudDir)
+        print("Just made new directories")
+    except FileExistsError:
+        print("Directories already exist")
+    
+    print("Location of console directory: \n%s\n" %consoleDir)
+    print("Location of cloud directory: \n%s\n" %cloudDir)
+
+    # before we get ahead of ourselves, we need to make sure that the necessary file
+    #   types are allowed (meaning, we are able to read them in)... in this example,
+    #   at the very least we need to have access to dicom and txt file types.
+    #   INPUT: None
+    #   OUTPUT:
+    #       [1] allowedFileTypes (list of allowed file types)
+    
+    allowedFileTypes = fileInterface.allowedFileTypes()
+    print(""
+    "-----------------------------------------------------------------------------\n"
+    "Before continuing, we should check to see the file types that are allowed.\n"
+    "To verify, we will use 'allowedFileTypes'. Only these files will be uploaded\n"
+    "to the cloud in the next step!!\n"
+    "Allowed file types: %s" %allowedFileTypes)
+
+    # we want to upload files from the 'console computer' to the cloud
+    # to do this, we will use 'uploadFilesToCloud' from 'projectUtils'
+    #   INPUT: 
+    #       [1] fileInterface (this allows us to use useful functions)
+    #       [2] srcPattern (the file pattern for the source directory)
+    #       [3] outputDir (the directory where you want the files to go)
+    srcPattern = os.path.join(consoleDir,'**')
+    projUtils.uploadFilesToCloud(fileInterface,srcPattern,cloudDir)
+
+    print(""
+    "-----------------------------------------------------------------------------\n"
+    "INITIALIZATION COMPLETE!")
 
 
 def main(argv=None):
-    logger = logging.getLogger()
-    logger.setLevel(logLevel)
-
+    """
+    This is the main function that is called when you run 'intialize.py'.
+    Here, you will set up an important argument parser (mostly provided by 
```

Comment:
Maybe revise "set up an important argument parser (mostly provided by the toml configuration file)"
to "load the configuration settings specified in the toml configuration file"